assets/js/blocks/rating-filter/block.json

```diff
@@ -1,8 +1,8 @@
 {
 	"name": "woocommerce/rating-filter",
 	"version": "1.0.0",
-	"title": "Filter by Rating Controls",
```

Comment:
We use the `Controls` suffix to differentiate the block with the controls from the wrapper block. Otherwise the structure is a bit confusing:

![imatge](https://user-images.githubusercontent.com/3616980/206692121-12dcd29f-d543-43d1-a115-546ef7fce89c.png)
_(the last one is the one that had the `Controls` suffix)_

@vivialice are we sure we want to go ahead with this change? Other filter blocks also have the `Controls` suffix in the inner block to differentiate it from the wrapper:

![imatge](https://user-images.githubusercontent.com/3616980/206692321-5dbc8a25-885e-40a8-82f3-e98717478e2f.png)